python/lsst/validate/drp/matchreduce.py

```diff
@@ -46,7 +44,7 @@
 
 
 def build_matched_dataset(repo, dataIds, matchRadius=None, safeSnr=50.,
-             useJointCal=False):
+                          useJointCal=False, skipTEx=False):
```

Comment:
Do you really want to force skipTEx to False here?

Reply:
Default for it is False everywhere, which is changed by the config option.